Add Swiftfin TVOS version to client codec support table (#1237)
jellyfin/jellyfin.org: docs/general/clients/codec-support.md

[Test your browser's compatibility for any codec profile.](https://cconcolato.github.io/media-mime-support/)

> | Sorted by efficiency (excluding bit depth)                                                                          | Chrome         | Edge           | Firefox | Safari          | Android        | Android TV     | iOS            | SwiftFin (iOS)  | [Roku](https://developer.roku.com/docs/specs/media/streaming-specifications.md) | Kodi | Jellyfin Media Player | SwiftFin (tvos)

Reviewer: ```suggestion
| Sorted by efficiency (excluding bit depth)                                                                          | Chrome         | Edge           | Firefox | Safari          | Android        | Android TV     | iOS            | SwiftFin (iOS)  | [Roku](https://developer.roku.com/docs/specs/media/streaming-specifications.md) | Kodi | Jellyfin Media Player | SwiftFin (tvOS)
```

| ------------------------------------------------------------------------------------------------------------------ | -------------- | -------------- | ------- | --------------- | -------------- | -------------- | -------------- | --------------- | ------------------------------------------------------------------------------- | ---- | --------------------- | ---------- |
| [MPEG-4 Part 2/SP](https://en.wikipedia.org/wiki/DivX)                                                             | ❌             | ❌             | ❌      | ❌              | ❌             | ❌             | ❌             | ✅              | ✅                                                                              | ✅   | ✅                    |      ✅       |
| [MPEG-4 Part 2/ASP](<https://en.wikipedia.org/wiki/MPEG-4_Part_2#Advanced_Simple_Profile_(ASP)>)                   | ❌             | ❌             | ❌      | ❌              | ❌             | ❌             | ❌             | ✅              |                                                                                 | ✅   | ✅                    |      ✅       |

> | [H.264 8Bit](https://caniuse.com/#feat=mpeg4 'H264 Browser Support Reference')                                     | ✅             | ✅             | ✅      | ✅              | ✅             | ✅             | ✅             | ✅              | ✅                                                                              | ✅   | ✅                    |       ✅   <sup>15</sup>      |

Reviewer: ```suggestion
| [H.264 8Bit](https://caniuse.com/#feat=mpeg4 'H264 Browser Support Reference')                                     | ✅             | ✅             | ✅      | ✅              | ✅             | ✅             | ✅             | ✅              | ✅                                                                              | ✅   | ✅                    |      ✅      |
```

If a codec is mentioned without the bitrate, then the standard is meant - so 8 bit.
Therefore its not needed to ad this comment to the codec standard.

> | [H.264 10Bit](https://caniuse.com/#feat=mpeg4 'H264 Browser Support Reference')                                    | ✅             | ✅             | ❌      | 🔶<sup>12</sup> | ✅             | ✅             | ❌             | ✅              | ❌                                                                              | ✅   | ✅                    |       ✅   <sup>15</sup>      |

Reviewer: ```suggestion
| [H.264 10Bit](https://caniuse.com/#feat=mpeg4 'H264 Browser Support Reference')                                    | ✅             | ✅             | ❌      | 🔶<sup>12</sup> | ✅             | ✅             | ❌             | ✅              | ❌                                                                              | ✅   | ✅                    |      🔶<sup>15</sup>      |
```

We cant just assume the codec is supported when not specified in the client documentation/ tested.
Therefore we will need to mark this as something "inbetween" working and not working.

> | [H.265 8Bit](https://caniuse.com/#feat=hevc 'HEVC Browser Support Reference')                                      | 🔶<sup>8</sup> | ✅<sup>7</sup> | ❌      | 🔶<sup>1</sup>  | 🔶<sup>2</sup> | ✅<sup>5</sup> | 🔶<sup>1</sup> | ✅<sup>6</sup>  | 🔶<sup>9</sup>                                                                  | ✅   | ✅                    |       ✅   <sup>15</sup>      |

Reviewer: ```suggestion
| [H.265 8Bit](https://caniuse.com/#feat=hevc 'HEVC Browser Support Reference')                                      | 🔶<sup>8</sup> | ✅<sup>7</sup> | ❌      | 🔶<sup>1</sup>  | 🔶<sup>2</sup> | ✅<sup>5</sup> | 🔶<sup>1</sup> | ✅<sup>6</sup>  | 🔶<sup>9</sup>                                                                  | ✅   | ✅                    |      ✅      |
```

As mentioned above, when not specified differently, we assume 8bit codecs are meant. Therefore there is no need to refer to the note here.

> | [H.265 10Bit](https://caniuse.com/#feat=hevc 'HEVC Browser Support Reference')                                     | 🔶<sup>8</sup> | ✅<sup>7</sup> | ❌      | 🔶<sup>1</sup>  | 🔶<sup>2</sup> | 🔶<sup>5</sup> | 🔶<sup>1</sup> | ✅<sup>6</sup>  | 🔶<sup>9</sup>                                                                  | ✅   | ✅                    |       ✅   <sup>15</sup>      |

Reviewer: ```suggestion
| [H.265 10Bit](https://caniuse.com/#feat=hevc 'HEVC Browser Support Reference')                                     | 🔶<sup>8</sup> | ✅<sup>7</sup> | ❌      | 🔶<sup>1</sup>  | 🔶<sup>2</sup> | 🔶<sup>5</sup> | 🔶<sup>1</sup> | ✅<sup>6</sup>  | 🔶<sup>9</sup>                                                                  | ✅   | ✅                    |      🔶<sup>15</sup>      |
```
We cant just assume the codec is supported when not specified in the client documentation/ tested.
Therefore we will need to mark this as something "inbetween" working and not working.

Author: also fixed in latest commits

<br />
<sup>11</sup>AV1 decoding is only available on devices with A17 or M3 series chips or newer and requires at least Safari 17.
<br />
<sup>12</sup>Need to be manually enabled in Settings > Playback > Enable H.264 High 10 Profile. Playback on Apple Silicon Macs with macOS version < 14 and Intel Macs with all macOS versions may result in blank frames if this is enabled.<br />

> <sup>13</sup> The official codec support [documentation](https://github.com/jellyfin/Swiftfin/blob/main/Documentation/players.md) does not mention bitrates only codecs

Reviewer 2: ```suggestion
<sup>13</sup>The official codec support [documentation](https://github.com/jellyfin/Swiftfin/blob/main/Documentation/players.md) does not mention bitrates only codecs
```

<br />
<sup>14</sup>AV1 is enabled by default for Swiftfin (VLCKit). AV1 is disabled by default but can be enabled for Native (AVKit) using Custom Device Profiles. Enabling AV1 may result in a poor experience for SOCs prior to A17.
<br />

> <sup>15</sup>The official codec support [documentation](https://github.com/jellyfin/Swiftfin/blob/main/Documentation/players.md) does not mention bitrates, only codecs

Reviewer: ```suggestion
<sup>15</sup>The official codec support [documentation](https://github.com/jellyfin/Swiftfin/blob/main/Documentation/players.md) does not mention support for 10bit encodings.
```

Maybe it makes sense to updade the wording here(?)
However I see this one as optional.

Reviewer 3: bit depth not bitrate

Reviewer 3: ```suggestion
<sup>15</sup>The official codec support [documentation](https://github.com/jellyfin/Swiftfin/blob/main/Documentation/players.md) does not mention bit depths, only codecs
```
